packages/types/src/definitions/runtime/Extratimer.type.ts

```diff
@@ -1,7 +1,9 @@
 export type SimplePlayback = 'play' | 'pause' | 'stop';
+export type SimpleDirection = 'up' | 'down';
```

Comment:
Suggestion: how about something more semantic of timers, like 
`type: 'count-down' | 'count-up'`

It achieves the same but has a similar wording to the other timers, which may make it less confusing for us